docs/index.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v1
 entries:
   argus:
   - apiVersion: v2
-    created: "2020-09-23T11:34:13.057324+05:30"
+    created: "2020-10-02T01:35:41.476376+02:00"
     description: Automated Kubernetes monitoring
-    digest: 200d6d8c89c79768bdd466c6872e0e27c469340992a9a91a48d146afc008f7f0
+    digest: 4b45cd6d11531d2470c651d1ea5d461272d5b36493b861e31f03ca12330f90b7
```

Comment:
Why is this touching the other packages?

Reply:
Makefile command `charts` updates index of all charts.
digest is used for verifying the integrity of helm charts.